docs/python/README.rst

```diff
@@ -52,6 +52,11 @@ replaces *scikit-learn* to compute the predictions.
 Changes
 -------
 
+0.3.0
+^^^^^
+
+C-API, Linux support for Dotnet Nuget package, Cuda 10.0 support (patch to 0.2.0).
```

Comment:
we don't support cuda 10 for this release. we're still going to be releasing with cuda 9.1.

Reply:
done.